apps/client-web/src/docs/pages/DocumentPage.tsx

```diff
@@ -39,11 +39,12 @@ export const DocumentPage: React.FC<DocumentPageProps> = ({ mode }) => {
 
   useDocHistoryProvider(docMeta.id as string)
 
-  const { provider, initBlocksToEditor, loading, awarenessInfos } = useBlockSyncProvider({
+  const { provider, initBlocksToEditor, loading, blockCommitting, awarenessInfos } = useBlockSyncProvider({
     blockId: docMeta.id as string,
     historyId: docMeta.historyId
   })
 
+  isSavingVar(blockCommitting.current)
```

Comment:
We should put this line into `useEffect()` (consider making `blockCommitting` a state instead of a ref). Otherwise it causes issues:

<img width="1326" alt="image" src="https://user-images.githubusercontent.com/5107241/174927349-e93bb69d-a607-4865-9776-1c39de24cd2d.png">

Reply:
https://github.com/mashcard/mashcard/pull/78